### ISSUE TITLE
Update to latest version of nodejs module

### ISSUE THREAD

**Issue by TraGicCode** [frozenfoxx/puppet-hubot#4]
Is it possible for you to try and get this working with the latest version of nodejs.  Currently we are passing the flag to prevent nodejs install but puppet module install is complaining since we have other modules that use the newer versions of nodejs module.

```
2018-03-08 06:16:45,325 WARN  [qtp1770542989-657673] [puppetserver] Puppet ModuleLoader: module 'hubot' has unresolved dependencies - it will only see those that are resolved. Use 'puppet module list --tree' to see information about modules
```

**Comment by frozenfoxx** [comment]
I've had some issues with getting NodeJS to behave in the past myself. On Ubuntu in particular I had to do the same thing since even when it was new the NodeJS module would remove the install (see [246](https://github.com/voxpupuli/puppet-nodejs/issues/246)).

I don't mind accepting pull requests but I don't use Hubot anymore (I use Errbot). As a result it's more difficult for me to test reliably. I don't mind accepting pull requests and can see if there's something I can do about upgrading the NodeJS version used but I'm really not familiar anymore with the current needs of Hubot.

**Comment by TraGicCode** [comment]
I have experience with getting this to work with the latest version of node.  Are you able to get on puppets public slack so we can chat?



Sent via the Samsung GALAXY S® 5, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by frozenfoxx** [comment]
Yes, I have joined the puppetcommunity slack, I'm @frozenfoxx on there, please feel free to ping me. I may not be always available but I'll chat as soon as I can.
